File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int largest = 0;
        int count = 0;
        for(int i = 0 ; i < nums.size() ; i++){
            if(nums[i]==0){
                count = 0;
            }else{
                count++;
                largest = max(largest,count);
            }
        }
        return largest;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int max(int x,int y){
        if(x>=y) return x;
        return y;
    }
    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int maximum = 0;
        for(int i = 0 ; i < nums.length ; i++){
            if(nums[i]==1) count++;
            else count = 0;
            maximum = max(count,maximum);
        }
        return maximum;
    }
}